.coveragerc

```diff
@@ -6,6 +6,7 @@ source =
     tests/unit
 omit =
     lms/scripts/*
+    lms/migrations/*
```

Comment:
Something about the code below causes coverage to notice `lms/migrations/`, which it didn't notice before, and fail because the migration scripts and co don't have test coverate